src/main/java/org/jabref/gui/collab/stringadd/StringAddDetailsView.java

```diff
@@ -0,0 +1,27 @@
+package org.jabref.gui.collab.stringadd;
+
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+
+import org.jabref.gui.collab.ExternalChangeDetailsView;
+import org.jabref.logic.l10n.Localization;
+
+public final class StringAddDetailsView extends ExternalChangeDetailsView {
+
+    public StringAddDetailsView(StringAdd stringAdd) {
+        VBox container = new VBox();
+        Label header = new Label(Localization.lang("Added string"));
+        header.getStyleClass().add("sectionHeader");
+        container.getChildren().addAll(
+                header,
+                new Label(Localization.lang("Label") + ": " + stringAdd.getAddedString().getName()),
+                new Label(Localization.lang("Content") + ": " + stringAdd.getAddedString().getContent())
```

Comment:
I'm not sure what our policy is with Labels and variable text. @Siedlerchr it seems odd that way